cmd/libsnap-confine-private/mount-opt.c

```diff
@@ -115,3 +118,134 @@ const char *sc_mount_opt2str(unsigned long flags)
 	}
 	return buf;
 }
+
+char *sc_mount_cmd(const char *source, const char *target,
+		   const char *filesystemtype, unsigned long mountflags,
+		   const void *data)
+{
+	// NOTE: this uses static buffer because it has lower complexity than a
+	// dynamically allocated buffer. We've decided as a team to prefer this
+	// approach.
```

Comment:
As discussed on IRC, there was a misunderstanding. The team was not advocating a static local variable, they were advocating a statically sized array (ie, a stack variable). While I don't have any issues with properly coded dynamic allocations, since this is cleared up please remove 'static char *buf[BUF_SIZE]' and simply use 'char buf[BUF_SIZE]' as described.